client/src/index.jsx

```diff
@@ -13,15 +13,20 @@ class App extends React.Component {
       isLoading: true
```

Comment:
First of all, I like your code very much. As always I noticed again that you are a very organized person. I have a question/comment in client/src/index.jsx: I am not very sure what is the correct thing to do. I see that you wrote your App in this file. I guess you notice it when you review my code that I created two files for index.jsx and app.jsx.

Reply:
Thank you! 
I initially separate index & app, then merged. You brought up a good point that it is separated in the style guide, so I'll make this change to meet style guide formatting.